identity/app/views/fragments/profile/membershipForm/upsell.scala.html

```diff
@@ -0,0 +1,8 @@
+<div class="membership-up-sell is-hidden js-mem-up-sell">
+    <div class="membership-up-sell__heading">
+        <h2 class="membership-header">Join Guardian Membership</h2>
+    </div>
+    <p class="membership-up-sell__content">We're bringing the Guardian to life through live events and meet-ups Join and be part of the conversations that matter</p>
```

Comment:
> Looks to me like "Join" should be on a new line?

according to @davidmcdowell it isn't - will confirm with him when he's back
![screen shot 2015-02-11 at 15 16 38](https://cloud.githubusercontent.com/assets/2305016/6149480/14808948-b201-11e4-9413-485ed9cbe2f3.png)